src/lib.rs

```diff
@@ -26,7 +26,7 @@
 //! and it gets more powerful every day!
 //!
 //! ### This Crate
-//! The `bevy` crate is just a container crate that makes it easier to consume Bevy components.
+//! The `bevy` crate is just a container crate that makes it easier to consume Bevy subcrates.
```

Comment:
This one-off change is to avoid confusion between the abstract meaning of "components" and the ECS meaning by being more specific.